rules.md

```diff
@@ -3,104 +3,131 @@ layout: default
 
-Sign-ups have started! Register on [FindCollabs](https://findcollabs.com/hackathon/codevid-19-isp21fkqtjupchx7kjed) and [join](https://join.slack.com/t/codevid-19/shared_invite/zt-cs8amank-jg7vUQeSUgX7K9cM9WZMfQ) our [Slack](https://codevid-19.slack.com). We will close sign-ups one week prior to the [final submission date](#submission).
+How we better understand and manage the economic impacts, and use our resources most effectively to weather the largest economic change in modern history.
 
-#### Do you have datasets and other resources we could start from?
+### Your Solution
 
-Volunteers are busy working on several items right now, and we're sorry we don't have a pre-compiled list for you. That said, there's been several posted in our [Slack](https://codevid-19.slack.com) workspace by our community. Click [here](https://join.slack.com/t/codevid-19/shared_invite/zt-cs8amank-jg7vUQeSUgX7K9cM9WZMfQ) to join!
+We believe faster, more effective and more available solutions are preferred to slower, less useful and limited ones.
 
-#### Will all projects be judged? Can I opt not to have my project submitted?
+Therefore we will be judging projects on three main criteria:
 
-Absolutely! We ask that teams who want to compete for prizing to submit their projects on their own.
+1. **Effectiveness** - How well does this project address the problem it's attempting to solve?
+2. **Acessability** - How easily are people are able to access this project?
+3. **Impact** - How many people have benefitted from this project?
 
-#### What's the cost of entry?
+Technilogical novelty and achievement, while impressive, won't be the determining factors when evaluating your project so remember to keep it simple!
 
-This hackathon is free to enter and particpate in! We thank our sponsors for supporting our endeavours.
+### Your Team
 
-#### Okay, so what's the catch?
+Teams of up to 5 people are welcome to participate. For legal reasons we require that all team members be 13 years or older as of May 16th at 00:00 UTC if they wish to participate.
 
-Participants and volunteers must abide by and uphold the [Code of Conduct](code-of-conduct.html), [Rules](#rules-submission-faq) and generally be an upstanding human being.
+### Prizes
 
-#### If I see organizers give out a time/deadline, what time zone is it in?
+Prizes of monetary value will be distributed to winning teams electronically. All other prizes will be disttributed in the manner deamed most appropriate by organizers and sponsors.
 
-Double-check with them, but we are running in UTC. Check [here](https://www.timeanddate.com/worldclock/) to see what that is relative to your local time.
+Some sponsors may be given the opportunity award prizes during the course of the competition. These prizes will be communicated and awarded by sponsors in the manner they sit fit, and subject to the rules and veto of the event organizers.
```

Comment:
"they sit fit" -> "they see fit", I make these mistakes all the time and it sucks because spellcheckers can't catch 'em :(